docker/Dockerfile

```diff
@@ -78,7 +78,8 @@ ADD dist /opt/spark/dist
 ADD krb5.conf.mustache /etc/
 
 # Add external jar for pushing metrics via StatsD (https://github.com/vidhyaarvind/spark-statsd)
-ADD jars/spark-statsd-1.0.0.jar /opt/spark/dist/jars/
+# current version: Spark 2.2.1, Scala 2.11
+ADD jars/spark-statsd-2.2.1-2.11.jar /opt/spark/dist/jars/  
```

Comment:
Does it make sense to fetch this using the Mesos fetcher instead? (maybe that is out of scope for this PR though).

Reply:
Yeah, maybe in a different PR but I'm more of the mind 'if it ain't broke don't fix it'